src/config.rs

```diff
@@ -204,6 +226,22 @@ fn parse_optional_size(val: &str) -> Result<Option<u64>> {
     })
 }
 
+fn verify_mount_point(val: &str) -> Result<PathBuf> {
+    let path = PathBuf::from(val);
```

Comment:
This likely wants to be `Path::new(val)`, since that won't allocate.

Reply:
`::new(val)` fails with `expected 0 arguments`.

Reply:
Hmm, this compiles
```diff
diff --git src/config.rs src/config.rs
index 135ced8001..84a0679d11 100644
--- src/config.rs
+++ src/config.rs
@@ -228,7 +228,7 @@ fn parse_optional_size(val: &str) -> Result<Option<u64>> {
 }
 
 fn verify_mount_point(val: &str) -> Result<PathBuf> {
-    let path = PathBuf::from(val);
+    let path = Path::new(val);
 
     if path.is_relative() {
         return Err(anyhow!("mount-point {} is not absolute", val));
@@ -238,7 +238,7 @@ fn verify_mount_point(val: &str) -> Result<PathBuf> {
         return Err(anyhow!("mount-point {:#?} is not normalized", path));
     }
 
-    Ok(path)
+    Ok(path.to_path_buf())
 }
 
```
I'm not sure if that's better.

Reply:
Well, I'm unsure on why you're actually returning it from here, but I did miss that you are. In this case, then, yeah, the `PathBuf::from()` variant is better.